src/front/spv.rs

```diff
@@ -1208,14 +1237,23 @@ impl<I: Iterator<Item = u32>> Parser<I> {
                 name: decor.name,
                 binding,
                 ty,
+                offset: decor
+                    .offset
+                    .ok_or(Error::MissingDecoration(spirv::Decoration::Offset))?,
             });
         }
         let inner = crate::TypeInner::Struct { members };
+        let decor = self.future_decor.remove(&id);
+        if let Some(ref decor) = decor {
+            if decor.block.is_some() {
+                // do nothing
+            }
+        }
```

Comment:
What is the reason for doing this, as the code does not do anything with it?

Reply:
the decoration applies to that type, so it needs to be used here, somehow
I just haven't figured out how exactly :)